### PR TITLE
[Docs] - pd2lork/pd - README.md - minor tweaks (spelling, grammar, syntax, writing)

### PR DESCRIPTION
**Pull request by ghost**
## [Docs] - pd2lork/pd - README.md - minor tweaks (spelling, grammar, syntax, writing)

### CODE REVIEW

**Comment by jonwwilkes** [comment]
I cherry picked some of these for Purr Data:

https://git.purrdata.net/jwilkes/purr-data/merge_requests/151

However, a lot of these are wrong. For example, "structure elements" means the fields contained inside a C struct. Changing that to "structural elements" makes it ambiguous and prone to misunderstanding.

In other areas you pick a singular verb like "is" for a plural subject like "patches." That would make the sentence grammatically incorrect.

Finally, in the part in "Coding Style" regarding compiler warnings your fix changes the meaning of verb pass from "succeed" to "generate". What Miller is saying there is that Pd has a coding style which, on the whole, *should not generate warnings to the compiler*. I can tell you that a large percentage of the warnings generated in Purr Data are actually subtle runtime bugs-- some innocuous, some not-- which should be fixed (the others are unused variables which are a different problem).

In other words, changing "of" to "to" there isn't a minor tweak, but a change in the content of the README. Such a proposed change should at least have its own pull request and should really start as a mailing list discussion to make sure everybody understands the coding style the same way.

**Comment by jonwwilkes** [comment]
Also-- it appears we had already fixed some of these in Purr Data (definite article in front of "Version", replacing some of the tabs with spaces, etc.)
